.travis.yml

```diff
@@ -37,8 +38,9 @@ env:
     - UNENCRYPTED_PACKAGE=true TARGET_PLATFORM=debian/jessie
     - UNENCRYPTED_PACKAGE=true TARGET_PLATFORM=ubuntu/bionic
     - UNENCRYPTED_PACKAGE=true TARGET_PLATFORM=ubuntu/xenial
+    - UNENCRYPTED_PACKAGE=true TARGET_PLATFORM=ubuntu/focal
```

Comment:
nit, I do not like the ordering of these distros. I guess we should group by family first, and in each group order them by the release date in descending order.

We can keep them as is, no big issue